index.bs

```diff
@@ -7623,9 +7853,14 @@ partial dictionary MLOpSupportLimits {
   </tr>
   <tr>
     <td>{{falseValue}}</td>
-    <td>[=/same as=] {{trueValue}}</td>
+    <td>[=/same type as|same as=] {{trueValue}}</td>
     <td>[=/any rank|N=]</td>
   </tr>
+  <tr>
+    <td>*output*</td>
+    <td>[=/same type as|same as=] {{trueValue}}</td>
+    <td>maximum of {{trueValue}}'s [=MLOperand/rank=] and {{falseValue}}'s [=MLOperand/rank=]</td>
```

Comment:
`condition`'s rank should also be included according to the algorithm step

```
1. Set |outputShape| to the result of [=bidirectionally broadcasting=] |condition|'s [=MLOperand/shape=] and |outputShape].
```

BTW, there is a typo of "|outputShape]" in the algorithm step, it should be "|outputShape|".

```suggestion
    <td>maximum of {{condition}}'s [=MLOperand/rank=], {{trueValue}}'s [=MLOperand/rank=] and {{falseValue}}'s [=MLOperand/rank=]</td>
```

Reply:
Fixed the typo in ab327cae116144297287bb66b1d32348380c1c64